src/api/api.c

```diff
@@ -197,6 +197,337 @@ int8_t zp_config_insert(z_loaned_config_t *config, uint8_t key, const char *valu
 #if Z_FEATURE_ENCODING_VALUES == 1
 #define ENCODING_SCHEMA_SEPARATOR ';'
 
+const z_owned_encoding_t ENCODING_ZENOH_BYTES = {
```

Comment:
It would be nice to define some macro to avoid repeating the definition code so many times.
And also use a separate file to avoid overloading api.c